src/localLighthouse.js

```diff
@@ -143,11 +163,7 @@ export default async ({
   let index = 1;
 
   for (const url of urls) {
-    if (verbose) {
-      console.log(`${NAME}: Auditing (${index}/${urls.length}) ${url}`);
-    }
-
-    const lighthouseAuditResult = await localLighthouse({
+    let options = {
```

Comment:
This should be a `const` (sorry the linting isn't better)

Reply:
Yes, will do.